### PR TITLE
Group integration module

### PR DESCRIPTION
**Pull request by AaronGilMartinez**

### CODE REVIEW

File: collabora_online.info.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Collabora Online
 description: Integrate Collabora Online for document viewing and editing.
-package: Custom
+package: Collabora online
```
Comment (donquixote) [inline collabora_online.info.yml]: Should we capitalize the "Online"? No opinion myself.

File: composer.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
         "drupal/core-composer-scaffold": "^10.3.6",
         "drupal/core-dev": "^10.3.6",
         "drupal/core-recommended": "^10.3.6",
+        "drupal/groupmedia": "^4",
```
Comment (donquixote) [inline composer.json:22]: Should we rely on indirect dependencies, or better require drupal/group too?

File: modules/collabora_online_group/collabora_online_group.info.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name: Collabora Online Group
+description: Integrates Collabora Online with Group module
+package: Collabora online
+dependencies:
+  - collabora_online:collabora_online
+  - groupmedia:groupmedia
```
Comment (donquixote) [inline modules/collabora_online_group/collabora_online_group.info.yml:6]: Same here about indirect dependencies.

File: phpunit.xml.dist [hunk 4]
```diff
@@ -8,6 +8,7 @@
   <php>
     <ini name="error_reporting" value="32767"/>
     <ini name="memory_limit" value="-1"/>
+    <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>
```
Comment (donquixote) [inline phpunit.xml.dist]: This change should be a separate commit imo.
Ideally rebase onto the other PR #6

File: composer.json (hunk 2 above)
Comment (donquixote) [reply composer.json:22]: I have not really evaluated the groupmedia module.
I would say keep it in this first version of the epic branch, and evaluate it further as we add more features.
Worst case is we remove it later.
Unless you want me to test something specific @AaronGilMartinez

File: collabora_online.info.yml (hunk 1 above)
Comment (brummbar) [reply collabora_online.info.yml]: Yep it's capitalised https://www.drupal.org/project/collabora_online

File: composer.json (hunk 2 above)
Comment (brummbar) [reply composer.json:22]: Indirect dependencies are fine in this case, as it's a mandatory dependency. Groupmedia works with group, it's a mandatory dependency. It would be different in case of a library.

Comment (AaronGilMartinez) [reply composer.json:22]: Changed

File: collabora_online.info.yml (hunk 1 above)
Comment (AaronGilMartinez) [reply collabora_online.info.yml]: Changed

File: composer.json (hunk 2 above)
Comment (AaronGilMartinez) [reply composer.json:22]: Ok

File: phpunit.xml.dist (hunk 4 above)
Comment (AaronGilMartinez) [reply phpunit.xml.dist]: Removed, created a separated commit for this

**Review by brummbar — APPROVED** [review]
